apps/backoffice-v2/src/pages/Entity/components/Case/hooks/usePendingRevisionEvents/utils/calculate-pending-workflow-events.ts

```diff
@@ -0,0 +1,27 @@
+import { TWorkflowById } from '@/domains/workflows/fetchers';
+import { StateTag } from '@ballerine/common';
+import { calculateWorkflowRevisionableEvent } from '@/pages/Entity/components/Case/hooks/usePendingRevisionEvents/utils/calculate-workflow-revisionable-event';
+import { IPendingEvent } from '@/pages/Entity/components/Case/hooks/usePendingRevisionEvents/interfaces';
+
+export const calculatePendingWorkflowEvents = (workflow: TWorkflowById): Array<IPendingEvent> => {
+  return workflow.context.documents
+    .map(document => {
+      if (!!document.decision.status && workflow?.tags?.includes(StateTag.MANUAL_REVIEW))
+        return {
+          workflowId: workflow.id,
+          documentId: document.id as string,
+          pendingEvent: calculateWorkflowRevisionableEvent(workflow, document.decision.status),
+          token: workflow?.context?.metadata?.token,
+        };
+    })
```

Comment:
If you don't want an early return here at least add curly braces for the if statement. 🙏🏼